### PR TITLE
Make Makefile great again

### PR DESCRIPTION
**Pull request by nerett**
- updated `Makefile`
- deleted `compile.sh`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
-CFLAGS =
+CC = clang++
+INCLUDE = -I../ELFIOgit
+CFLAGS = -Wno-initializer-overrides -Wno-c99-designator $(INCLUDE)
 BUILDDIR = Build
 
-all: mkdir
-	$(CC) $(CFLAGS) main.cpp functions.cpp instructions.cpp -o $(BUILDDIR)/simulator
+XCC = riscv64-elf-gcc
+XCFLAGS = -nostdlib -march=rv64g
```
Comment (vian96) [inline Makefile:7]: isn't it supposed to be `-march=rv64i`?

Comment (nerett) [reply Makefile:7]: For some reason `riscv64-elf-gcc` says that `addi` is somehow connected with `d` extension
image

Comment (nerett) [reply Makefile]: Anyway, you're right, fixed `Makefile`

**Comment by vian96** [comment]
LGTM, merging

**Review by vian96 — APPROVED** [review]

**Pull request merged**